server/src/test/java/org/apache/druid/server/log/FilteredRequestLoggerTest.java

```diff
@@ -106,6 +123,9 @@ public void testNotFilterAboveThreshold() throws IOException
     EasyMock.expect(nativeRequestLogLine.getQueryStats())
             .andReturn(new QueryStats(ImmutableMap.of("query/time", 1000)))
             .once();
+    EasyMock.expect(nativeRequestLogLine.getQuery())
```

Comment:
@leventov Does that wrapping indentation of 4 standard include lines such as these 3? I see that the lines above and below are had been using 8.

Reply:
No. Here, the alignment is on the dot. It's a coincidence that "EasyMock" is 8 characters long.